csrc/quantization/awq/gemm_kernels.cu

```diff
@@ -90,7 +90,7 @@ __global__ void __launch_bounds__(64) gemm_forward_4bit_cuda_m16n128k32(int G, i
                             + (((int)threadIdx.x) % (128 / 8)) * 8;
 
   half* C_ptr = C 
-              + blockIdx_z * M * OC        // blockIdz.x -> split_k dim
+              + (long)blockIdx_z * M * OC        // blockIdz.x -> split_k dim
```

Comment:
> style nit: Can we use `static_cast<long long>` instead? I think it would make the code clearer.

Changed. The file itself contains lots of C-style cast though.

Reply:
Thanks!